content/1_General/Debugging_Lang.mdx

```diff
@@ -343,45 +419,15 @@ failure?
 		url="https://codeforces.com/blog/entry/15547"
 		starred
 	>
-		includes all the options mentioned below
+		Includes all the options mentioned below.
 	</Resource>
 </Resources>
 
-You can also check what options
-[Errichto](https://codeforces.com/blog/entry/64993) and
-[ecnerwala](https://gist.github.com/ecnerwala/a3c6332ac626bc448165) use.
-
 ### Warning Options
 
-<Resources>
-	<Resource
-		starred
-		source="GCC"
-		url="https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html"
-		title="3.8 - Options to Request or Suppress Warnings"
-	>
-		documentation for options below
-	</Resource>
-</Resources>
-
-Such as the following:
-
-```
--Wall -Wextra -Wshadow -Wconversion -Wfloat-equal -Wduplicated-cond -Wlogical-op
```

Comment:
^ leave this in too

Reply:
wait what is this even supposed to tell people about though

Reply:
some of the most helpful warning options

Reply:
ok i'm gonna put that in the resource description itself then

Reply:
actually i looked at it, and it just gives a "look these are the things we're gonna go over"
i feel like that's accomplished by just scrolling through the headers?

Reply:
wdym? these options aren't in the headers

Reply:
alr so these options are just "these are some extra options that might be cool"?

Reply:
yes